Source/AppScaffolding/LibationScaffolding.cs

```diff
@@ -86,110 +85,6 @@ public static void RunPostConfigMigrations(Configuration config)
 			Migrations.migrate_to_v6_6_9(config);
 		}
 
-		public static void PopulateMissingConfigValues(Configuration config)
-		{
-			config.InProgress ??= Configuration.WinTemp;
-
-            if (!config.Exists(nameof(config.UseCoverAsFolderIcon)))
-                config.UseCoverAsFolderIcon = false;
-
-            if (!config.Exists(nameof(config.BetaOptIn)))
```

Comment:
It feels like the right move to get these values out of scaffolding and into their own domain areas. I didn't check each of these default throughout your PR but I trust that you did

Reply:
I did. One by one I deleted them from scaffolding and added the default value into configuration.